sat-manpage.sh

```diff
@@ -26,8 +26,14 @@
 
 # Text surrounded by @ is replaced by a sed command in the spec file
 sat_repository=${SAT_REPOSITORY:-@DEFAULT_SAT_REPOSITORY@}
-sat_image_tag=${SAT_TAG:-@DEFAULT_SAT_TAG@}
-sat_image=${SAT_IMAGE:-$sat_repository:$sat_image_tag}
+if [[ -f "/etc/sat-version" ]]; then
```

Comment:
Do we have a convention around dropping arbitrary files in `/etc`? I personally don't have a problem with this but I wonder if CSM or MTL folks would rather it go somewhere like `/etc/opt/cray/sat` or something like that.

Reply:
Something like `/etc/opt/cray/sat/version` might just be better, honestly, I'll give that a go.

Reply:
I addressed this in 71a6ffc28ae7a4c91a0b2c42ad859dd6ba1e3709

Reply:
I noticed that `sat-podman` implicitly expects `/opt/cray/etc/` to exist when starting the container (the former location of the "local release files") and the local "cache" location of the site info file. Maybe it would be better to use `/opt/cray/etc` as we'd be indirectly addressing that issue.

I happened to notice that SDU's install seems to place things in `/etc/opt/cray/sdu` while I see a `cray-orca` directory in `/opt/cray/etc/`.

Reply:
`/etc/opt/cray` feels a little more consistent with the [Filesystem Hierarchy Standard](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) IMO. I would consider this a configuration file.

Reply:
I don't feel strongly about this though.

Reply:
Reading over that document is helpful. I can't figure out why we would have chosen to use `/opt/cray/etc/` in the first place.

I also don't feel too strongly other than I like the idea that our configuration creates the "site file cache" directory now if it does not already exist, so I'm just going to leave this I think.